app/src/main/java/com/hover/stax/bounties/SectionedBounty.java

```diff
@@ -0,0 +1,41 @@
+package com.hover.stax.bounties;
+
+import com.hover.stax.channels.Channel;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+class SectionedBounty {
```

Comment:
This object already exists, it is a Channel. This duplicates objects and data in a way that will cause bugs. Use or directly connect to an existing object rather than duplicating information from it.